### PR TITLE
Bug 2004585: prometheus-k8s-0 cpu usage keeps increasing for the first 3 days

### PR DESCRIPTION
**Pull request by sanchezl**
- split slos alerts & records into basic & extended groups
- skip registering extended slos alerts on SNO

### CODE REVIEW

File: bindata/assets/alerts/kube-apiserver-slos-basic.yaml [hunk 1]
```diff
@@ -167,40 +139,6 @@ spec:
       labels:
         verb: read
       record: apiserver_request:burnrate1h
-    - expr: |
```
[the remark this answers is not in the document]
Comment (tkashem) [reply bindata/assets/alerts/kube-apiserver-slos-basic.yaml:170]: i think the expression is what's spiking the cpu usage

File: bindata/assets/alerts/kube-apiserver-slos-extended.yaml [hunk 2]
```diff
@@ -0,0 +1,189 @@
+apiVersion: monitoring.coreos.com/v1
```
Comment (tkashem) [inline bindata/assets/alerts/kube-apiserver-slos-extended.yaml:1]: so this is just the original file copied into a new name, right?

**Review by tkashem — COMMENTED** [review]
looks good to me, let's see how much it improves SNO.

File: bindata/assets/alerts/kube-apiserver-slos-extended.yaml (hunk 2 above)
Comment (sanchezl) [reply bindata/assets/alerts/kube-apiserver-slos-extended.yaml:1]: The original file contents were distributed to two separate files, one of which (`extended`) is skipped on SNO installs.

File: bindata/assets/alerts/kube-apiserver-slos-basic.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanchezl) [reply bindata/assets/alerts/kube-apiserver-slos-basic.yaml:170]: Just to close out this conversation: the trace provided in the BZ did show that it was the actual evaluation of the recording rules that were causing the load, not the alerts themselves.

**Pull request merged**